Makefile

```diff
@@ -0,0 +1,41 @@
+.PHONY: configure build debug run test push push-image
+
+IMAGE_NAME ?= k8s-crds
+IMAGE_PREFIX ?= jsonnet-libs
+IMAGE_TAG ?= 0.0.1
+
+INPUT_DIR ?= ${PWD}/libs/k8s-alpha
+OUTPUT_DIR ?= ${PWD}/gen
+
+ABS_INPUT_DIR := $(shell realpath $(INPUT_DIR))
+JSONNET_FILE := $(ABS_INPUT_DIR)/config.jsonnet
+ABS_OUTPUT_DIR := $(shell realpath $(OUTPUT_DIR))
+
+configure:
+	jsonnet -c -m $(ABS_INPUT_DIR) -S $(JSONNET_FILE)
```

Comment:
I would add a comment to state that this requires `go-jsonnet` and not `jsonnet`